buildbot/configure.py

```diff
@@ -6,6 +6,29 @@
 import sys
 
 
+class _DeprecatedCmakeOpt(argparse._AppendAction):
+    sleep = 15
+
+    def __call__(self, parser, namespace, values, option_string):
+        print("""
+        ****
+        the `--cmake-opt` argument is deprecated and will be removed in a future
+        version. It is no longer necessary to prefix cmake-options
+        ****
```

Comment:
How about printing this message after cmake output instead of sleeping for 15 seconds?